packages/protocol/test/common/multisig.ts.old

```diff
@@ -133,6 +133,10 @@ contract('MultiSig', (accounts: any) => {
     it('should not allow an external account to add an owner', async () => {
       await assertRevert(multiSig.addOwner(accounts[3]))
     })
+
+    it('should not allow a new owner - the 0x address to be added', async () => {
```

Comment:
For consistency with our other tests: "it should not allow adding the null address"